core/urls.py

```diff
@@ -18,6 +20,9 @@
 
         # Stock price JSON data
         url(r'view/stockprice/$', views.stockPrices, name='stockPrice'),
+
+        # Leaderboard api
+        url(r'^api/leaderboard/$', views.leaderboardApi, name='leaderboardApi'),
```

Comment:
This is not required. We don't need to expose the leaderboard api before the event starts.

Reply:
views should have a database check which contains their state.. that is enabled or disabled. accordingly the views should return the result or 404. I dont know any cleaner way to enable and disable views.